linked_list.py

```diff
@@ -0,0 +1,77 @@
+class Node(object):
+    def __init__(self, data, next_node=None):
+        self.data, self.next = data, next_node
+
+
+class LinkedList(object):
+    """Create and object-->linked_list."""
+    def __init__(self, head=None):
+        self.head = head
+
+    def insert(self, data):
+        """Put the given value into Node at head of list."""
+        inserted_node = Node(data)
+        inserted_node.next = self.head
+        self.head = inserted_node
```

Comment:
You don't need all three lines here.  Given the constructor you built for the `Node` class, you can do this method in one line of code (and you should).